src/storage/storage_info.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace duckdb {
 
-const uint64_t VERSION_NUMBER = 42;
+const uint64_t VERSION_NUMBER = 43;
```

Comment:
I think the storage version does not need to be increased here. The storage version only has to be increased if backwards incompatible changes are made to the reader. In this case you only need to re-generate the storage version as you added an entry to the `test_all_types` function.

Reply:
Ah ok I misunderstood that, I've reverted it now!